modular-docs-manual/content/topics/module_modular-documentation-repositories.adoc

```diff
@@ -0,0 +1,141 @@
+        ├── ...
+    ├── ...
+├── titles-enterprise
+    ├── kogito-configuring
+        ├── _images -> ../../_images
+        ├── assemblies -> ../../assemblies
+        ├── master.adoc
+        └── master-docinfo.xml
+    ├── kogito-creating-running
+        ├── ...
+    ├── kogito-developing-decision-services  // Contains multiple assemblies.
+        ├── ...
+    ├── ...
+├── pantheon2.yml
+└── README.md
+----
+
+This repository structure contains the following key components:
+
+* `_artifacts`: Directory with document attribute files and other metadata files. The underscore in the folder name is an optional indication of a non-content folder.
+* `_images`: Directory with image files, optionally divided into sub-directories by subject matter. The underscore in the folder name is an optional indication of a non-content folder.
+* `assemblies`: Directory with assembly files. You collect and compile assemblies into documents (titles) using the `titles-enterprise` folder.
+* `modules`: Directory with module files, optionally divided into sub-directories by subject matter.
+* `titles-enterprise`: Directory with folders for each enterprise document or _title_ to be built in Pantheon. Each document folder contains a `master.adoc` file where you include one or more relevant assemblies that are built as part of the document. The `master-docinfo.xml` file contains metadata that is required by `bccutil` (and `ccutil`) when converting to DocBook format. (This metadata file might be deprecated in a future release of `bccutil` and Pantheon 2.) In repositories that are single sourced with a community project, a separate `titles-community` folder might be dedicated to community configurations and output, if applicable.
+* `pantheon2.yml`: YAML file with metadata that is required for publishing content on Pantheon 2:
++
+.Example `pantheon2.yml` file
+[source,yaml]
+----
+server: http://localhost:8080
+repository: mod-doc-repo-example
+variants:
+   - name: global-attributes
+     path: _artifacts/document-attributes.adoc
+     canonical: true
+   - name: community-attributes
+     path: _artifacts/document-attributes-kogito-comm.adoc
+   - name: enterprise-attributes
+     path: _artifacts/document-attributes-kogito-ent.adoc
+assemblies:
+  - assemblies/*.adoc
+modules:
+  - modules/*/*.adoc
+resources:
+  - _images/*/*.png
+  - _artifacts/*.adoc
+----
++
+For information about configuring this file for your specific content, contact a member of the CCS tooling team.
+
+This repository structure offers the following potential advantages:
+
+* *Flexibility in determining publishable titles.* In this structure, the images, artifacts, assemblies, and modules all serve as distinct building blocks that you use to compile a document or _title_. You can include an image in multiple modules, a module in multiple assemblies, and an assembly in multiple titles. Additionally, just as an assembly can include multiple modules, a title can contain multiple assemblies.
```

Comment:
```suggestion
* *Flexibility in determining publishable titles.* In this structure, the images, artifacts, assemblies, and modules all serve as distinct building blocks that you use to compile a document or _title_. You can include an image in multiple modules, a module in multiple assemblies, and an assembly in multiple titles. Additionally, just as assemblies can include multiple modules, titles can contain multiple assemblies.
```

Reply:
Plural "assemblies" containing multiple modules and "titles" containing multiple assemblies doesn't give the emphasis of multiple in the same assembly/title that we're trying to convey. Two titles each with one assembly still = plural titles with multiple assemblies. I was trying to emphasize multiple in one.

Boy, I'm sure being stiff on this PR so far but really not trying to :( I can give up and go with the flow if I need to. :)

Reply:
2 in 1 shampoo never made sense to me. 1 isn't big enough to hold 2. That's why there's 2.

Jokes aside, it's fine if you don't want to make the change. I generally prefer using plural forms for consistency. Some day I'll have to explain the reason behind that but I can't put it into words at the moment.

Reply:
Haha! Good one :)  Well I'll look forward to that rationale (honestly, not being sarcastic at all). Sounds interesting. For now, will leave as is, if okay with you.